digicampipe/tests/test_pulse_template_object.py

```diff
@@ -13,28 +13,54 @@
         'pulse_SST-1M_pixel_0.dat'
     )
 )
+template_filename_2 = resource_filename(
+    'digicampipe',
+    os.path.join(
+        'tests',
+        'resources',
+        'template_all.txt'
+    )
+)
+
+
+dir_data = '/mnt/baobab/sst1m/raw/2018/05/'
```

Comment:
I suppose these should be removed

Reply:
as you wanted to start from the histogram, I replaced it by the histogram in 2fab9c5